src/state/user.js

```diff
@@ -24,6 +26,16 @@ const loadUser = async () => {
   return user;
 };
 
+const loadGuidedTours = async () => {
+  const { accessToken } = useAuth();
```

Comment:
AccessToken can be managed directly in the service, see https://github.com/bimdata/platform-next/blob/develop/src/services/BcfService.js#L32